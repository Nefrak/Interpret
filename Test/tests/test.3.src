.IPPcode18
DEFVAR GF@counter
MOVE GF@counter string@mmm
PUSHS string@ccc
PUSHS GF@counter
WRITE string@counter\032obsahuje\032
SETCHAR GF@counter int@1 string@k
WRITE GF@counter
POPS GF@counter
WRITE GF@counter
POPS GF@counter
WRITE GF@counter
WRITE string@\010
LABEL end