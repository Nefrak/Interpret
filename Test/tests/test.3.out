counter obsahuje 
mkm
mmm
ccc



